test/single/data/expected_buildkite_pipeline.yaml

```diff
@@ -1000,6 +1000,22 @@ steps:
     automatic: true
   agents:
     queue: cpu
+- label: ':fire: MPI PyTorch MNIST (test-cpu-mpich-py3_8-tf2_4_1-keras2_4_3-torch1_7_1-mxnet1_7_0_p2-pyspark_3_0_1)'
```

Comment:
here are the new tf2 runs of this example

Reply:
I think this is okay, we should make the decision based on the version of PyTorch, not TF.